DTS-STN/Service-Canada-Labs
# Splash page styling

# Description

The purpose of this pr is to update the styling for the splash page. In my previous pr, there are some styling issues for the background image and the main cta card. The main issue was the background image being cut off when the screen resolution gets bigger, this should be fixed in this pr. 

I also updated the styling for the main cta to match the new design on Figma. Updated the background to use a slightly darker color instead of pure white, and applied box shadow. 

## Acceptance Criteria

Splash page matches the new design on [Figma](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?node-id=10436%3A27681)

## Test Instructions

1. yarn dev
2. visit the splash page (make sure to clear cookies, otherwise you'll be redirected to the home page)
3. check the styling

## Checklist
